resolve/matrix/Sparse.cpp

```diff
@@ -199,6 +299,15 @@ namespace ReSolve { namespace matrix {
     }
   }
 
+  /**
+   * @brief updata matrix values using the _new_values_ provided either as HOST or as DEVICE array. This function will copy the data (not just assign a pointer) and allocate if needed. It also sets ownership and update flags.
+   *
+   * @param[in] new_vals    - pointer to new values data (array of real numbers)
```

Comment:
Separate brief from long description:
```suggestion
  /**
   * @brief updata matrix values using the _new_values_ provided either as HOST or as DEVICE array. 
   *
   * This function will copy the data (not just assign a pointer) and allocate if needed. It also sets ownership and update flags.
   *
   * @param[in] new_vals    - pointer to new values data (array of real numbers)
```